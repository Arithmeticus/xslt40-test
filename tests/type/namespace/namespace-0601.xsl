<?xml version="1.0"?> 
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">

<!-- Test copying of namespace nodes -->
<!-- REF: XSLT 1.0 Errata -->
  <?spec xslt#copy-of?>

<xsl:template match="/">
<out>
<a1><xsl:copy-of select="//q/namespace::*"/></a1>;
<a2><xsl:copy-of select="//q/namespace::a"/></a2>;
<a3><xsl:for-each select="//namespace::*"><xsl:copy/></xsl:for-each></a3>;
</out>
</xsl:template>
 
</xsl:stylesheet>
