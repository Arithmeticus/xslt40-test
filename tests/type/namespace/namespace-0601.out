<?xml version="1.0" encoding="UTF-8"?><out><a1 xmlns:a="http://a.uri/" xmlns:c="http://c.uri/" xmlns:b="http://b.uri/"/>;
<a2 xmlns:a="http://a.uri/"/>;
<a3 xmlns:a="http://a.uri/" xmlns:c="http://c.uri/" xmlns:b="http://b.uri/"/>;
</out>